rcsb/db/wf/RepoLoadWorkflow.py

```diff
@@ -313,7 +326,14 @@ def splitIdList(self, op, **kwargs):
                 # Split up single holdings file into multiple sub-lists
                 holdingsFile = os.path.join(holdingsFileBaseDir, list(holdingsFileD.keys())[0])
                 hD = mU.doImport(holdingsFile, fmt="json")
-                idL = [k.upper() for k in hD]
+                if useImgsFormat:
+                    idL = self.getCsmImgsFormattedList(
+                        hD,
+                        updateAllImages=kwargs.get("updateAllImages"),
+                        BcifBaseDir=kwargs.get("BcifBaseDir"),
```

Comment:
I think you missed updating `BcifBaseDir` here. (BTW, this is another reason for moving the kwarg extraction to the start of the method, so that you don't have to modify it in so many places)